assets/report/input.css

```
@@ -147,7 +147,29 @@
 
   /* Sidebar navigation (shadcn sidebar/tabs idiom) */
   .report-nav {
-    @apply sticky top-8 hidden h-[calc(100vh-4rem)] w-56 shrink-0 flex-col gap-1 overflow-y-auto pr-2 lg:flex;
+    @apply sticky top-8 hidden h-[calc(100vh-4rem)] w-56 shrink-0 flex-col gap-1 overflow-y-auto overscroll-contain pb-6 pr-2 lg:flex;
+    scroll-behavior: smooth;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)